testing/polkit-qt5/APKBUILD

```diff
@@ -0,0 +1,35 @@
+# Contributor: Bart Ribbers <bribbers@disroot.org>
+# Maintainer: Bart Ribbers <bribbers@disroot.org>
+pkgname=polkit-qt5
```

Comment:
Again, this should be `polkit-qt`, to match upstream.  It won't ever be built against Qt4.